### PR TITLE
DIGI_29_UniqueTaxPageTitle

### PR DESCRIPTION
**Pull request by rajendarreddy1234**
1.Added TaxPageController
2.Added TaxPage/Create.cshtml

### CODE REVIEW

File: DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj [hunk 1]
```diff
@@ -116,6 +116,7 @@
     <Compile Include="App_Start\BundleConfig.cs" />
     <Compile Include="App_Start\FilterConfig.cs" />
     <Compile Include="App_Start\RouteConfig.cs" />
+    <Compile Include="Controllers\AppliesController.cs" />
```
Comment (prakashkolli) [inline DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj:119]: check applies controller.....are u adding Applies controller??

File: DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj [hunk 2]
```diff
@@ -261,6 +262,7 @@
     <Content Include="Views\Category\Create.cshtml" />
     <Content Include="Views\TaxPage\Create.cshtml" />
     <Content Include="Views\TaxPage\Index.cshtml" />
+    <Content Include="Views\Applies\Index.cshtml" />
```
Comment (prakashkolli) [inline DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj:265]: check once applies index view

File: DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj (hunk 1 above)
Comment (rajendarreddy1234) [reply DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj]: Resolved

File: DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj (hunk 2 above)
Comment (rajendarreddy1234) [reply DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj]: Resolved

**Pull request merged**